src/genie/libs/parser/iosxe/tests/test_show_crypto_ikev2_sa_detail.py

```diff
@@ -0,0 +1,234 @@
+# Import the Python mock functionality
```

Comment:
This file can be removed.

Reply:
I can do that.  Are these files never needed in a PR?  The docs seemed to seem like they where but I got confused with the docs also saying "the old way"